portfolio/src/main/webapp/js/custom-script.js

```diff
@@ -423,3 +425,62 @@ function drawInteractiveChart() {
         interactiveChart.draw(emotionData, options);
       });
 }
+
+// TODO(briafassler): Don't leave function here
+typewriterFeature();
+
+function typewriterFeature() {
+  const typedText = document.querySelector('.typed-text');
+  /**
+   * TODO Stretch Feature(briafassler): read names from a database instead of an
+   * array
+   */
+  const martyrNames = [
```

Comment:
make the variable name all caps to be consistent with contstants

Reply:
done